prisma/seed.ts

```diff
@@ -52,6 +100,37 @@ async function main() {
             },
             update: {},
         }),
+        prisma.authUser.upsert({
+            where: {
+                id: 'test_f5ec79b4-8c49-4015-bc37-f2758923ef38',
+            },
+            create: {
+                id: 'test_f5ec79b4-8c49-4015-bc37-f2758923ef38',
+                firstName: 'Karan',
```

Comment:
I think we should probably make the name something generic.

Reply:
Done.